turbo.json

```diff
@@ -24,6 +24,10 @@
       "dependsOn": ["build"],
       "outputs": []
     },
+    "profile": {
+      "dependsOn": ["@quri/squiggle-lang#build"],
+      "outputs": []
```

Comment:
I haven't checked this carefully, but I expect this to break CI on PRs that don't re-run the profile due to turbo repo caches.

Or maybe CI just won't store the new profile then, since `upload-artifact` relies on glob? Then maybe it's ok and would save us some space in remote caches.